src/test/groovy/org/prebid/server/functional/tests/module/pbrequestcorrection/PbRequestCorrectionSpec.groovy

```diff
@@ -0,0 +1,455 @@
+package org.prebid.server.functional.tests.module.pbrequestcorrection
+
+import org.prebid.server.functional.model.config.AccountConfig
+import org.prebid.server.functional.model.config.AccountHooksConfiguration
+import org.prebid.server.functional.model.config.PbRequestCorrectionConfig
+import org.prebid.server.functional.model.config.PbsModulesConfig
+import org.prebid.server.functional.model.db.Account
+import org.prebid.server.functional.model.request.auction.BidRequest
+import org.prebid.server.functional.model.request.auction.AppExt
+import org.prebid.server.functional.model.request.auction.AppPrebid
+import org.prebid.server.functional.model.request.auction.Device
+import org.prebid.server.functional.model.request.auction.Imp
+import org.prebid.server.functional.model.request.auction.OperationState
+import org.prebid.server.functional.service.PrebidServerService
+import org.prebid.server.functional.tests.module.ModuleBaseSpec
+import org.prebid.server.functional.util.PBSUtils
+
+import static org.prebid.server.functional.model.request.auction.DistributionChannel.APP
+import static org.prebid.server.functional.model.request.auction.OperationState.YES
+
+class PbRequestCorrectionSpec extends ModuleBaseSpec {
+
+    private static final String PREBID_MOBILE = "prebid-mobile"
+    private static final String DEVICE_PREBID_MOBILE_PATTERN = "PrebidMobile/"
+    private static final String ACCEPTABLE_DEVICE_UA_PREBID_MOBILE_VERSION = PBSUtils.getRandomVersion("0.0", "2.1.5")
+    private static final String ACCEPTABLE_DEVICE_INSTL_PREBID_MOBILE_VERSION = PBSUtils.getRandomVersion("0.0", "2.2.3")
+    private static final String ANDROID = "android"
+    private static final String IOS = "IOS"
+
+    private PrebidServerService pbsServiceWithRequestCorrectionModule = pbsServiceFactory.getService(requestCorrectionSettings)
+
+    def "PBS should remove positive instl from imps for android app when request correction is enabled for account"() {
+        given: "Android APP bid request with version lover then 2.2.3"
```

Comment:
We should avoid directly mentioning the version. It will result in less work in the future :)
`given: "Android APP bid request with version lover then version threshold"`